### PR TITLE
SCSS: don't add extra comma after last comment in map

### PR DESCRIPTION
**Pull request by fisker**
fixes: #6911

FYI:

```scss
$my-map: (
  "foo": 1,
  // Comment
    "bar": 2,
  //        ^ <----- this PR don't remove this extra comma, when trailingComma is none
  // Comment,
  //        ^ <----- this PR only removed this extra comma
);
```

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by fisker** [comment]
As I said, this PR only fix comma after the last comment. #6920 is another issue

File: changelog_unreleased/scss/pr-6918.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+5. Optionally write a description. Many times it’s enough with just sample code.
+
+6. Change ```jsx to your language. For example, ```yaml.
+
+7. Change the `// Input` and `// Prettier` comments to the comment syntax of your language. For example, `# Input`.
+
+8. Choose some nice input example code. Paste it along with the output before and after your PR.
+
+-->
```
Comment (lydell) [inline changelog_unreleased/scss/pr-6918.md]: Please remove this comment. It’ll make creating a changelog entry / blog post much easier.

Comment (fisker) [reply changelog_unreleased/scss/pr-6918.md]: done

**Pull request merged**